doc/user/content/get-started/quickstart.md

```diff
@@ -24,13 +24,13 @@ the superpowers of an operational data warehouse first-hand:
 
 * **Consistency**: results are always correct; never even transiently wrong.
 
-## Before you begin
+## Prerequisite
 
-All you need is a Materialize account. If you already have one —
-great! If not, [sign up for a playground account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation) first.
+A Materialize account to sign into the [Materialize console](https://console.materialize.com/). If you do not have an account, you can [sign up for a free trial account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
 
-When you're ready, head over to the [Materialize console](https://console.materialize.com/),
-and pop open the SQL Shell.
+## Step 0. Sign into the Console
+
+Open the [Materialize console](https://console.materialize.com/) and, if not already signed in, sign in. By default, the SQL Shell should display in the console.  You can also select SQL Shell in the console's left-hand menu.
```

Comment:
```suggestion
Navigate to the [Materialize console](https://console.materialize.com/) and sign
in. By default, you should land in the SQL Shell. If you're already signed in,
you can access the SQL Shell in the left-hand menu.
```

Reply:
If a user signs in for the first time, they'll **always** land in the shell (which might change in a few quarters), so it feels more relevant to just point already signed-in users to the SQL Shell if they're somewhere else!